### ISSUE TITLE
Visual Glitch: Information Overlay Is Not Updated When Multiple Actions Are Performed

### ISSUE THREAD

**Issue by blankster** [thesobercoder/polygon#4]
**Describe the bug**
If several quick actions are performed succeeding, the information on the overlay is not updated correctly.

**To Reproduce**
1. Execute multiple actions succeeding
2. The information overlay shows only the result of the first action.

**Expected behavior**
The information overlay always displays the result of the last action, without the need to wait when the information disappears from the first action.

**Screenshots**
-

**Operating System**
Provide the output from `systeminfo`.

For example:
```
OS Name:                   Microsoft Windows 11 Pro
OS Version:                23H2 (10.0.22631.2715)
```

**Comment by thesobercoder** [comment]
@blankster This is fixed in v0.5.0. Please get the latest version and try it out.

**Comment by thesobercoder** [comment]
@blankster This is actually a very tricky problem. The way it works is if there is already an overlay window, Polygon will first try to close that window first and then show the next one. Obviously, I'm aware of that this is not the most elegant fix. I will try and see if I can find out a better solution.
